appinfo/info.xml

```diff
@@ -34,7 +34,7 @@ The rating depends on the installed text processing backend. See [the rating ove
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
 	]]></description>
-	<version>4.0.0-alpha.3</version>
+	<version>5.0.0-dev.0</version>
```

Comment:
Check with Christoph if we want to go with 5.0.0 or 4.1.0

Reply:
4.1 as long as we keep 30 as min